tests/Unit/Fixer/NotificationUserNoteAssetsUrlFixerTest.php

```diff
@@ -51,24 +51,24 @@ public function cheevosDataProvider(): array {
 
 		return [
 			[
-				'<img src="/extensions-ucp/v2/Cheevos/images/gp30.png"',
```

Comment:
**issue:** I think this paths should stay as it is (with `v2`) because there is a filter function that should change path from `v2` into `mw139`  placed here `\Reverb\Fixer\NotificationUserNoteAssetsUrlFixer` and this test is checking correctness of this filter